css/obBase.css

```diff
@@ -5684,6 +5751,13 @@ input[type="checkbox"].fieldItem:checked + label .togLabelOff {
   display: block;
 }
 
+#ov1 .marginCenter {
```

Comment:
No, the problem is that 'blockCenter' also adds `display: block` and then my adding of the `hide` class did nothing.